docs/releases/5.0.txt

```diff
@@ -358,3 +358,5 @@ to remove usage of these features.
 * Passing ``nulls_first=False`` or ``nulls_last=False`` to ``Expression.asc()``
   and ``Expression.desc()`` methods, and the ``OrderBy`` expression is no
   longer allowed.
+
+* The undocumented ``django.contrib.admin.helpers.checkbox`` is removed.
```

Comment:
This note should be moved to the _"Backwards incompatible changes in 5.0" -> "Miscellaneous"_ section, see [comment](https://github.com/django/django/pull/16515/files#r1101863924).